cmds/coredhcp/config.yml.example

```
@@ -156,3 +156,9 @@ server4:
         # * lease duration can be given in any format understood by go's
         # "ParseDuration": https://golang.org/pkg/time/#ParseDuration
         - range: leases.txt 10.10.10.100 10.10.10.200 60s
+
+        # optional static routes could be defined as well
+        # - staticroute: <destination>,<gateway> <destination>,<gateway>
```

nit: make it slightly more obvious that this is one or more entries:
```suggestion
        # - staticroute: <destination>,<gateway> [<destination>,<gateway> ...]
```